lookout/style/typos/preparation.py

```diff
@@ -40,6 +29,26 @@
     "raw_data_filename": "raw_data.csv",
     "vocabulary_filename": "vocabulary.csv",
     "frequencies_filename": "frequencies.csv",
+    "prepared_filename": "prepared.csv",
+}
+
+defaults_for_fasttext = {
+    "size": 100000000,
+    "corrupt": True,
+    "typo_probability": 0.2,
+    "add_typo_probability": 0.005,
+    "fasttext_path": str(DATA_DIR / "emb.bin"),
```

Comment:
Better to use long names: embeddings.bin

And btw what is `bin`? If it is the numpy format, we should use `npy`, etc.

Reply:
It's fasttext default format for saving full model.